CHANGELOG.md

```diff
@@ -6,13 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ## [Unreleased]
 
+## [0.5.0] - 26.12.2023
+
 ### Added 
 
-- Add new templates for `epic` issue and `feature request` - [#4](https://github.com/IronFoundation/iOS-Woof/pull/4)
-- Add the possibility for sitters to update info about themselves - [#52](https://github.com/IronFoundation/iOS-Woof/pull/52)
-- Add enums `AppButton` and `AppAlert` which contains repeating strings and text in buttons - [#55](https://github.com/IronFoundation/iOS-Woof/pull/55)
-- Add the possibility for owners to filter sitters by city - [#56](https://github.com/IronFoundation/iOS-Woof/pull/56)
-- Add the development server for test goals - [#57](https://github.com/IronFoundation/iOS-Woof/pull/57)
+- Added a city filter to the list of sitters - [56](https://github.com/IronFoundation/iOS-Woof/pull/56)
+- Add new templates for `epic` issue and `feature request` - [4](https://github.com/IronFoundation/iOS-Woof/pull/4)
+- Add the possibility for sitters to update info about themselves - [52](https://github.com/IronFoundation/iOS-Woof/pull/52)
+- Add enums `AppButton` and `AppAlert` which contain repeating strings and text in buttons - [#55](https://github.com/IronFoundation/iOS-Woof/pull/55)
```

Comment:
Why did you remove `#` when in the previous releases we use format with `#`
```
- Added a city filter to the list of sitters - [#56](https://github.com/IronFoundation/iOS-Woof/pull/56)
```

Reply:
It's just a typo.
Fixed with 9361860